packages/react-wildcat/src/config/wildcat.config.js

```diff
@@ -143,6 +143,11 @@ const wildcatConfig = {
                 //         this.body = "Hello from the server world!";
                 //     }));
                 // }
+                function(app) {
+                    app.use(
+                        require("../middleware/reactErrorOverlayEditorLauncher.js")()
```

Comment:
This can be overwritten upstream. This is a core function of Wildcat, so it belongs in either `packages/react-wildcat/src/server.js` or `packages/react-wildcat/src/staticServer.js`